test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -81,14 +66,21 @@ class test_uct_sockaddr : public uct_test {
         server_params.err_handler                    = err_handler;
         server_params.err_handler_arg                = reinterpret_cast<void*>(this);
         server_params.err_handler_flags              = 0;
-        server_params.mode.sockaddr.listen_sockaddr  = listen_sock_addr;
+        server_params.mode.sockaddr.listen_sockaddr  = m_listen_addr.to_ucs_sock_addr();
         server_params.mode.sockaddr.cb_flags         = UCT_CB_FLAG_ASYNC;
         server_params.mode.sockaddr.conn_request_cb  = conn_request_cb;
         server_params.mode.sockaddr.conn_request_arg = reinterpret_cast<void*>(this);
 
         server = uct_test::create_entity(server_params);
         m_entities.push_back(server);
 
+        uint16_t actual_port = ucs::sock_addr_storage(server->iface_params()
```

Comment:
why would the actual port be different than the one set in line 56? what could have changed it?

Reply:
because uct_iface_open can return UCS_ERR_BUSY if address in use and we need to pass another one at entity creation

Reply:
if BUSY returns, do we try to bind to another port?
I think it's worth to add a comment here stating the reason for the 'actual port'

Reply:
Ok, will do